_posts/current/blog/2020-09-02-linaro-engineering-highlights-for-august-2020.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: Linaro Engineering Highlights for August 2020
+description: This blog covers the many developments the teams at Linaro have
+  been working on during August including the initial TVM AI compiler
+  performance numbers on ARM64, using energy model to stay in TDP budget,
+  thermal notifications with Netlink, how the ARM32 Linux kernel decompresses
+  and the history of the Linux kernel and Linaro at one million commits.
+date: 2020-09-14T04:19:30.000Z
+image: /assets/images/content/10-year-graphic-horizonal1.png
+tags:
+  - XYZ
```

Comment:
Is this tag correct?

Reply:
No. This was a 'placeholder' so I could save the document. Will now add.

Thanks
Jon